scripts/zones/The_Garden_of_RuHmet/npcs/_0zy.lua

```diff
@@ -13,14 +13,14 @@ end
 
 entity.onTrigger = function(player, npc)
     -- the door breaks during the CS in Al'Taieu after receiving the Dawn mission, which sets the var to 1. Also checking for The Last Verse mission for whenever that gets implemented.
-    if ((player:getCurrentMission(xi.mission.log_id.COP) == xi.mission.id.cop.DAWN and player:getCharVar("PromathiaStatus") > 0) or player:getCurrentMission(xi.mission.log_id.COP) > xi.mission.id.cop.DAWN) then
+    if (player:getCurrentMission(xi.mission.log_id.COP) == xi.mission.id.cop.DAWN and player:getCharVar("PromathiaStatus") > 0) or player:getCurrentMission(xi.mission.log_id.COP) > xi.mission.id.cop.DAWN then
         if (player:getZPos() <= 360) then
             player:startEvent(140)
         else
             player:startEvent(141)
         end
     else
-        if (player:getZPos() <= 360) then
+        if player:getZPos() <= 360 then
```

Comment:
For some reason I at first thought this file was related to the Qn'Zdei but now I see it is not. Oh well it is just removing extra parenthesis.